scripts/generator/graphql_generator/csharp/SDK/Android/AndroidWebCheckout.cs.erb

```diff
@@ -0,0 +1,45 @@
+namespace <%= namespace %>.SDK.Android {
+    #if UNITY_ANDROID
+    using System;
+    using <%= namespace %>.SDK;
+
+    #if !SHOPIFY_MONO_UNIT_TEST
+    using UnityEngine;
+    #endif
+
+    class AndroidWebCheckout: WebCheckout {
+
+        private ShopifyClient _client;
+        private Cart _cart;
+
+        protected override ShopifyClient Client {
```

Comment:
In https://github.com/Shopify/unity-buy-sdk/blob/77c1288da163ab499637d02a96ae2c5869719f25/scripts/generator/graphql_generator/csharp/SDK/iOS/iOSWebCheckout.cs.erb#L15 it looks like ShopifyClient and Cart are just sitting public, whereas here they are private with accessors. Is there a reason we have differing access for these between Android and iOS?